ui/admin/app/routes/scopes/scope/session-recordings/index.js

```diff
@@ -4,5 +4,160 @@
  */
 
 import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+import { action } from '@ember/object';
 
-export default class ScopesScopeSessionRecordingsIndexRoute extends Route {}
+export default class ScopesScopeSessionRecordingsIndexRoute extends Route {
+  // =services
+  @service store;
+  @service router;
+  @service can;
+
+  // =attributes
+
+  queryParams = {
+    search: {
+      refreshModel: true,
+      replace: true,
+    },
+    times: {
+      refreshModel: true,
+      replace: true,
+    },
+    users: {
+      refreshModel: true,
+      replace: true,
+    },
+    scopes: {
+      refreshModel: true,
+      replace: true,
+    },
+    targets: {
+      refreshModel: true,
+      replace: true,
+    },
+    page: {
+      refreshModel: true,
+    },
+    pageSize: {
+      refreshModel: true,
+    },
+  };
+
+  allSessionRecordings;
+
+  /**
+   * Load all session recordings.
+   * @return {Promise<{ totalItems: number, sessionRecordings: [SessionRecordingModel], sessionRecordingsExist: boolean, storageBucketsExist: boolean }>}
+   */
+  async model({ search, times, users, scopes, targets, page, pageSize }) {
+    const scope = this.modelFor('scopes.scope');
+    const { id: scope_id } = scope;
+    let sessionRecordings;
+    let totalItems = 0;
+    let sessionRecordingsExist = false;
+    let storageBucketsExist = false;
+    const filters = {
+      scope_id: [{ equals: scope_id }],
+      created_time: [],
+      'create_time_values.user.id': [],
+      'create_time_values.target.scope.id': [],
+      'create_time_values.target.id': [],
+    };
+    times.forEach((time) => {
+      filters.created_time.push({ gte: new Date(time) });
```

Comment:
non-blocking: out of curisoity, is conversion to Date object necessary?

Reply:
I could have been doing something wrong with my dates and the format but I had to do this to get the filtering to work properly.

Reply:
Do you know if it's just a string or an actual date object in indexed db?

Reply:
Can we actually also add some tests using dates to `indexed-db-query-test.js` to confirm this behavior? This is the first time we've used dates

Reply:
It is stored as a date object in indexed db. I will work on some tests as well.